src/test/data/encodings/iso88591

```diff
@@ -18,3 +18,5 @@ About the Unicode Consortium
 The Unicode Consortium is a non-profit organization founded to develop, extend and promote use of the Unicode Standard, which specifies the representation of text in modern software products and standards. The membership of the consortium represents a broad spectrum of corporations and organizations in the computer and information processing industry. The consortium is supported financially solely through membership dues. Membership in the Unicode Consortium is open to organizations and individuals anywhere in the world who support the Unicode Standard and wish to assist in its extension and implementation.
 
 For more information, see the Glossary, Unicode Enabled Products, Technical Introduction and Useful Resources.
+
+Der schnelle braune Fuchs sprang �ber den faulen Hund.
```

Comment:
Can you please explain this change?

Reply:
ISO-8859-1 supports more than just English, this adds an ISO-8859-1 encoded German sentence. It appears as "�" in the diff as GitHub decodes it as UTF-8. Without this sentence, the file contains only ASCII characters.